<html xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns="http://www.w3.org/1999/xhtml"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>MIBBI_Foundry_compilation</title>
</head>
<body>
<table border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr>
<th align="left">
<h2>
<tt>&nbsp;MIBBI_Foundry_compilation</tt>
</h2>
</th>
</tr>
<tr>
<td>
<table align="left" bgcolor="#999999" border="0" cellpadding="5" cellspacing="1" width="100%">
<tbody><tr bgcolor="#00CC00">
<td width="2%"><b>Index</b></td><td width="18%"><b>Item name</b></td><td width="40%"><b>Definition</b></td><td width="40%"><b>Examples</b></td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>CIMR</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Mammalian and other in vivo experiments</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1</b>&nbsp;</td><td><b>Standards for mammalian pre-clinical studies</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.1</b>&nbsp;</td><td><b>Experimental subject description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Species or strain designation&nbsp;</td><td>For rat or mouse http://www.informatics.jax.org/mgihome/nomen/strains.shtml&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Generation of mixed strain&nbsp;</td><td>If different from 
[base] species and strain as given. While ‘generation of mixed strains’ 
is a requested ﬁeld, a number of physiological studies have found a 
profound inﬂuence on the phenotype of an animal during that study as a 
result of drift in the strain background.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Model description&nbsp;</td><td>Surgical, pharmacological or feeding manipulation&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Animal supplier&nbsp;</td><td>Company, location, colony designation if any, if wild caught&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Age range&nbsp;</td><td>Dates of birth, as well as ages at time of experiment&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Weight range&nbsp;</td><td>Give individual weights if available&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.2</b>&nbsp;</td><td><b>Husbandry</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.2.1</b>&nbsp;</td><td><b>Housing</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Group or individual&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Cage type&nbsp;</td><td>&nbsp;</td><td>shoe box, metabolic, wire mesh&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Cage change or cleaning frequency&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Environmental enrichment&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>1.2.2</td><td>Light cycle&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.2.3</b>&nbsp;</td><td><b>Feed</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Type and manufacturer&nbsp;</td><td>Or reference to composition if custom diet.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Ad lib or restricted diet&nbsp;</td><td>&nbsp;</td><td>25 grams per day&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Diet supplements&nbsp;</td><td>If any (what treats, how often, how much)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.2.4</b>&nbsp;</td><td><b>Water</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Bottle or automated&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tap or purified&nbsp;</td><td>Qualified&nbsp;</td><td>distilled, 18 MΩ&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>1.2.2</td><td>Veterinary treatments if any and exercise regimen (large animals)&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>1.2.6</td><td>Use of anesthesia&nbsp;</td><td>For example, for blood collection or physicals. [N.B. Numbering absent in manuscript -- corrected.]&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.2.7</b>&nbsp;</td><td><b>Acclimation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Acclimation duration&nbsp;</td><td>To experimental facility&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Acclimation duration&nbsp;</td><td>To diet (if experimental diet differs)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.3</b>&nbsp;</td><td><b>Experimental design</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.3.1</b>&nbsp;</td><td><b>Number of groups</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Numbers, gender and group sizes&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>1.3.2</td><td>Inclusion criteria&nbsp;</td><td>&nbsp;</td><td>physical exams, normal metabolomic model screen&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.3.3</b>&nbsp;</td><td><b>Treatments</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Compound&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Route&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Dose&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Dose volume&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Duration of dosing&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Vehicle&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>1.3.4</td><td>Fasting&nbsp;</td><td>When relative to metabonomic sample collection.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.3.5</b>&nbsp;</td><td><b>End Points</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Euthanasia method&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tissue collection list&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tissue processing method&nbsp;</td><td>&nbsp;</td><td>snap freezing&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Clinical signs&nbsp;</td><td>Time of observation relative to dose.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Body weights and food consumption&nbsp;</td><td>How often measured&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Blood chemistries, haematology, histopathology, special assays&nbsp;</td><td>The
 terms blood chemistry, haematology, histology and special assays may be
 populated in many studies, but these terms certainly will not be 
universal, especially in laboratories with limited resources, hence they
 were not deemed to be required information.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.4</b>&nbsp;</td><td><b>Metabolomics-related sample collection</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.4.1</b>&nbsp;</td><td><b>Blood</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Volume collected&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Location of collection&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Time of collection&nbsp;</td><td>Relative to dose and light cycle.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serum or plasma&nbsp;</td><td>Give anticoagulant or presence of serum separator.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.4.2</b>&nbsp;</td><td><b>Urine</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How collected&nbsp;</td><td>&nbsp;</td><td>metabolic cage, cystocentesis, catheterisation&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Frequency of collection&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Duration of collection&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Time of collection relative to dose and light cycle&nbsp;</td><td>If less than 24 hours&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Bacteriostatic agent or any other additive&nbsp;</td><td>ﬁnal concentration&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Urine volume&nbsp;</td><td>For 24 hour collections&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Temperature of urine collection tube&nbsp;</td><td>On ice/room temp&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1.4.3</b>&nbsp;</td><td><b>Tissues</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Identification&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Approximate quantity taken&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tissue processing method&nbsp;</td><td>&nbsp;</td><td>snap freezing, time from kill to snap freezing&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2</b>&nbsp;</td><td><b>Standards for mammalian clinical trials and human studies</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.1</b>&nbsp;</td><td><b>Experimental subject description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Ethical approval details&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Geographical location&nbsp;</td><td>If subjects are at a hospital please identify it.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Ethnic background&nbsp;</td><td>Based on FDA and Ofﬁce of National statistics criteria.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Demographics&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Medical history&nbsp;</td><td>Disease or clinical symptoms;
 criteria for disease presence (all volunteers should not have factors 
in their medical history which confound the study); e.g. surgical, 
pharmacological manipulation, medication (may be referenced).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Age range&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Weight range and height&nbsp;</td><td>May provide BMI as well or instead.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Gender&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Trial type&nbsp;</td><td>&nbsp;</td><td>randomised trial, disease biomarker, phase I-IV&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Diet and dietary restrictions&nbsp;</td><td>Describe relevant control groups for such dietary restrictions. Diet - standardized, isocaloric, free living subjects.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Further descriptors&nbsp;</td><td>&nbsp;</td><td>smoking, 
blood pressure, anomalies in habitual diet (e.g. vegetarian, vegan 
etc.), sporting activity and frequency, habitual alcohol 
consumption.&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.2</b>&nbsp;</td><td><b>Experimental design</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.2.1</b>&nbsp;</td><td><b>Number of groups</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Numbers, gender, group size&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>2.2.2</td><td>Inclusion criteria&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td>2.2.3</td><td>Exclusion criteria&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.2.4</b>&nbsp;</td><td><b>Treatment and fasting</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Compound&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Route&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Dose&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Dose volume&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Duration of dosing&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Vehicle&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.2.5</b>&nbsp;</td><td><b>End points</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Clinical chemistries, blood chemistry and haematology&nbsp;</td><td>&nbsp;</td><td>urea,
 creatinine, glucose, total cholesterol, HDL-cholesterol, 
LDL-cholesterol, triglycerides, total protein, albumin, erythrocyte 
count, haemoglobin, haemocrit, platelets, white blood count, sodium, 
potassium, bilirubin, ALT, ALP, -GT.&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Urine chemistry&nbsp;</td><td>&nbsp;</td><td>osmolality, ketones, pH, protein, glucose, bilirubin, blood, sediment and colour.&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.3</b>&nbsp;</td><td><b>Metabolomics-related sample collection</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.3.1</b>&nbsp;</td><td><b>Blood</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Volume collected&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Location of collection&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serum or plasma&nbsp;</td><td>Identify any anticoagulant; 
(serum only), time allowed for clotting and temperature; (plasma and 
serum), temperature of centrifugation, time and speed of 
centrifugation.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Arterial or venous blood collected&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Observations of haemolysis&nbsp;</td><td>In samples, with reporting of whether samples were used in subsequent analysis.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Time from separation to freezing/freezing process&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.3.2</b>&nbsp;</td><td><b>Urine</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Frequency and volume of collection&nbsp;</td><td>State whether mid-flow or total urine.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Bacteriostatic agent or any other additive&nbsp;</td><td>ﬁnal concentration&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2.3.3</b>&nbsp;</td><td><b>Tissues</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Identification&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Approximate quantity taken&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Post mortem tissues&nbsp;</td><td>&nbsp;</td><td>Hours after death until sample collection&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Microbial and in vitro biology experiments -- sample source</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Experimental design</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Biological question or the goal of the experiment&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Experimental design details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Samples&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Treatments or growth conditions&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Time or timing of sample collection&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Extracts&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Repeats&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Metabolome samples&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Phenotypic characteristics of the samples&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other 'omics' data sets generated from these same samples&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Biosource</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Source or supplier of the cell line or strain&nbsp;</td><td>&nbsp;</td><td>ATCC&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>In case of natural isolates</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>If at all possible, cultures should be deposited in an 
international culture collection with an accession number and relevant 
details.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>What taxonomic system was used to identify the (micro-)organism?&nbsp;</td><td>Because micro-organisms are often identified incorrectly.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>In case of mutant strains</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>From which wild-type were they obtained and how?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>In case of (higher) eukaryotes</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Cell type, organ derivation, grade of differentiation, subcellular location&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Immortalized or transformed&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Cell storage&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Growth environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Growth container: Type, supplier, geometry of the fermenter/bioreactor, (shake) flask or microtiter plates&nbsp;</td><td>Growth supports (type and supplier) in case of cells cultured in adherence.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Growth configuration&nbsp;</td><td>&nbsp;</td><td>suspension
 or attached culture, monolayer, double layer, sandwich, spheroids, 
batch, fed-batch, perfusion, continuous fermentation&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Inoculation procedure</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Subculturing and splitting protocols&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Inoculation size, seeding density&nbsp;</td><td>Inoculation
 size, seeding density (volume percentage [v.v^-1], num of cells per ml 
for suspension cultures; num of cells per cm^2 for cells in adherence, 
subconfluence or confluence.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Medium or substrates&nbsp;</td><td>Type and supplier, 
concentration or percentage -- including additions and supplementations 
(antibiotics, growth factors, serum type, and batches, concentration or 
percentage).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Environmental conditions&nbsp;</td><td>Which of the environmental conditions were controlled and which could alter (freely) during growth?&nbsp;</td><td>Temperature, pH, gas composition, humidity, percent CO2, stirrer speed, evaporation, pO2&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Growth rate&nbsp;</td><td>If the cells were grown at a set fixed growth rate: which was the growth rate?&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Treatment and incubation conditions</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Treatment factors</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Biotic&nbsp;</td><td>&nbsp;</td><td>competition with or infection by other organisms&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Abiotic&nbsp;</td><td>&nbsp;</td><td>physical stresses, chemical substances&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>treatment dose and vehicle&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Treatment time and intervals&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Labeling protocols for any use of or incubation with labeled substrates&nbsp;</td><td>13C compounds used (percent enrichment, purity)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Pretreatment&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Harvesting</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Biotic characteristics of moment of harvesting&nbsp;</td><td>&nbsp;</td><td>growth
 phase/stage (logarithmic, stationary, steady state, cytostatic phase, 
cell cycle phase, ...), number of generations in case of continuous 
cultures, time of sampling, stabilization time/phase before experiment, 
number of culture passages, independent indicators of differentiated 
state (immunological or molecular markers)&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Abiotic characteristics at time of harvesting&nbsp;</td><td>cell density [OD, DWT, counts], depletion of nutrients, treatment time&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Phenotypic characteristics especially relating to the question under study&nbsp;</td><td>&nbsp;</td><td>yield, productivity, color, form&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Microbial and in vitro biology experiments -- sample processing</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sampling</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>What is the time between sample removal from its environment until metabolic activity is truly stopped?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>What was the temperature during this process?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How were the samples harvested?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Quenching</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How was the metabolism of the samples shut down?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How is the cell integrity under the quenching conditions?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Extracellular metabolites</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How were intracellular metabolites discriminated from extracellular metabolites?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Extraction of metabolites from the cells</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How were the (intracellular) metabolites extracted from the cells?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>What is the estimated recovery at this step?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Normalization of the metabolome data</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How were the metabolome data normalized?&nbsp;</td><td>Specifically
 (in case of normalizing the data with respect to the amount of cells 
[no., mg] that they were obtained from), how and at what step was the 
amount of biomass determined?&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sample clean-up or work-up</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How were the samples cleaned-up with respect to compounds that interfere with analysis?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sample storage</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How and how long were the samples stored after collection, during work-up and prior to analysis?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Quality control steps</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>How was it verified that no biotic or abiotic changes 
occurred, or were at least minimized, during the complete sample 
collection and work-up phase?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Detection limit</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>What is the detection limit of the metabolites for the samples analyzed in the study?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Stability</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>What is known about the stability of (specific) metabolites during quenching, extraction and sample preprocessing?&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Environmental context</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>S</b>&nbsp;</td><td><b>Sample</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>S1</b>&nbsp;</td><td><b>Description of the biological sample involved in the study</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Taxonomic classification of organism(s)&nbsp;</td><td>Please
 give details of all organisms sampled, as far along the taxonomic scale
 as possible, ideally to the levels of genus, species and sub-species. 
Refer to a taxonomic classification, such as the NCBI taxonomy 
(http://www.ncbi.nlm.nih.gov/Taxonomy/) or the Integrated Taxonomic 
Information System (http://www.itis.gov/)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample composition&nbsp;</td><td>Please provide details of 
the organism(s) that constitute the sample. Amounts may be described in 
‘absolute’ (number of individuals) or relative terms (50% Organism X; 
25% Organism Y; 25% Unknown).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample Type&nbsp;</td><td>Please give details of the type of sample (for example; Community, Population, Whole Organism, Organ, Biofluid, etc.).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Condition of specimen(s)&nbsp;</td><td>Please give details of general observations on health, etc.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E</b>&nbsp;</td><td><b>Environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E1</b>&nbsp;</td><td><b>Description of ANY field environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Geographic location&nbsp;</td><td>Please specify latitude 
and longitude in decimal degrees. If relevant, you can also provide 
position in a local coordinate system e.g. the UK’s Ordnance Survey grid
 (http://www.ordnancesurvey.co.uk/)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Altitude or depth&nbsp;</td><td>Please specify in meters above/below sea level.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Habitat&nbsp;</td><td>Please provide a descriptor of habitat type.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Meteorological conditions&nbsp;</td><td>&nbsp;</td><td>weather type (sunny, snowing, etc.), humidity, precipitation, wind speed and direction&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Lunar or solar phase&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>All other measured parameters&nbsp;</td><td>&nbsp;</td><td>Pollutant concentration(s)&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E2</b>&nbsp;</td><td><b>Description of ANY laboratory environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Laboratory address and contact details&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E3.1</b>&nbsp;</td><td><b>Description of terrestrial environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Inclination and aspect&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Substrate type&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Substrate temperature&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>All other measured parameters&nbsp;</td><td>&nbsp;</td><td>substrate pH, substrate organic content&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E3.2</b>&nbsp;</td><td><b>Description of aquatic environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample(s) was submerged and emerged (how deep and for how long in this condition)&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Water temperature&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tidal phase&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>All other measured parameters&nbsp;</td><td>&nbsp;</td><td>pH, salinity, dissolved (in)organic content&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E3.3</b>&nbsp;</td><td><b>Description of atmospheric environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Atmospheric temperature&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>All other measured parameters&nbsp;</td><td>&nbsp;</td><td>atmospheric pressure, (in)organic content&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>E3.4</b>&nbsp;</td><td><b>Description of biotic environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of host organism&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Relationship of organism(s) to host&nbsp;</td><td>&nbsp;</td><td>parasite&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>All other measured parameters&nbsp;</td><td>&nbsp;</td><td>pH, temperature&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P</b>&nbsp;</td><td><b>Process</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P1</b>&nbsp;</td><td><b>Description of capture or sampling of sample or organism(s)</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of capture or sampling procedure&nbsp;</td><td>Include details such the method used&nbsp;</td><td>netted, electrically stunned, anaesthetised, razor cut&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reason for capture&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other capture parameters&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P2</b>&nbsp;</td><td><b>Description of storage or preservation of sample(s)</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of storage/preservation procedure&nbsp;</td><td>Include details such as the storage or preservation medium&nbsp;</td><td>liquid nitrogen, dry ice, formaldehyde&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reason for storage/preservation&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other storage or preservation parameters&nbsp;</td><td>&nbsp;</td><td>temperature&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P3</b>&nbsp;</td><td><b>Description of maintenance of organism(s)</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of maintenance procedure&nbsp;</td><td>Include details such as the type of housing, for example; Cage, aquaria, continuous culture, seed bag or plant pot, etc.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reason for maintenance&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other maintenance parameters&nbsp;</td><td>&nbsp;</td><td>feeding regime, cage dimensions&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P4</b>&nbsp;</td><td><b>Description of transportation of samples or organism(s)</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of transportation procedure&nbsp;</td><td>Include details such as the means of transport, for example; refrigerated container.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reason for transportation&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other transportation parameters&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P5</b>&nbsp;</td><td><b>Description of acclimation of organism(s)</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of acclimation procedure&nbsp;</td><td>Include details such as the type of housing, for example; cage, aquaria, continuous culture, seed bag or plant pot, etc.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reason for acclimation&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other acclimation parameters&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>P6</b>&nbsp;</td><td><b>Description of general manipulation of sample or organism(s)</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manipulation type&nbsp;</td><td>Perturbation&nbsp;</td><td>exposure to a toxicant, dissection, sacrifice&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reason for manipulation&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Description of manipulation procedure&nbsp;</td><td>&nbsp;</td><td>perturbation of specific environmental parameter, dissection of specific tissue&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other manipulation parameters&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Plant biology context</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1</b>&nbsp;</td><td><b>BioSource</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Species&nbsp;</td><td>Names of species should be described 
according to the NCBI taxonomy database (Wheeler et al. 2000; Benson et 
al. 2000) 
(http://pubmedexpress.nih.gov/Taxonomy/taxonomyhome.html/index.cgi). 
Plant species need to be named in full and not abbreviated, e.g. 
Arabidopsis thaliana. All necessary information on taxonomic 
relationships can be derived from the correct species name and thus does
 not need to be reported further.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Genotype&nbsp;</td><td>Subspecies information such as 
ecotype, cultivar and accession should be described according to 
authoritative databases such as TAIR (http://www.arabidopsis.org/). In 
the case of crosses or breeding results, available pedigree information 
must be given. In the case of transgenic or mutant organisms, name of 
the gene(s) that are up- or down-regulated should be reported, and the 
GenBank Accession number(s) for the sequence(s) of the corresponding 
construct(s), in addition to the parental subspecies background 
information, should be given. According to standard practice in 
agronomic genotype nomenclature, genotype description should comprise 
the author who first described or collected the cultivar, e.g. Medicago 
truncatula (Gaertn) cv. Jemalong. If available, registrations numbers 
for agronomic plants should be referenced, e.g. USDA GRIN. The number of
 backcrosses used in breeding needs to be detailed. In case of 
plant–pathogen interaction studies or other studies where information on
 multiple genomes is relevant, such metadata should be given.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Organ&nbsp;</td><td>Names of organs and plant structure 
should be described according to the authoritative database (Katica et 
al. 2007) maintained by the Plant Ontology Consortium to be found at 
http://www.plantontology.org/. All necessary information on organ 
relationships can be derived from the correct organ name and thus does 
not need to be reported further.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Organ specification&nbsp;</td><td>This should be provided 
only if such information cannot be detailed by 
http://www.plantontology.org/ (e.g. description of a part of an organ, 
the specific location of the organ or a specific tissue of an 
organ).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Cell type&nbsp;</td><td>This should be provided only if 
such information can be detailed in a meaningful manner, e.g. by cell 
sorting or dissection. Naming according to the authoritative database 
maintained by the Plant Ontology Consortium is to be found at 
http://www.plantontology.org/ under plant_structure ontology. Only if 
such information cannot be located at this source the Cell Ontology 
maintained at Open Biomedical Ontologies group should be taken, which is
 to be found at 
http://lists.sourceforge.net/lists/listinfo/obo-cell-type.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Subcellular location&nbsp;</td><td>This should be described
 only if such information can be detailed in a meaningful manner, e.g. 
by subcellular fractionation. Naming according to the authoritative 
database (Gene Ontology Cellular Component) maintained by the Gene 
Ontology Consortium to be found at http://www.geneontology.org/&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>BioSource amount&nbsp;</td><td>This refers to the mass (mg 
fresh weight or mg dry weight), number of cells or other measurable bulk
 quantities (e.g. protein content).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2</b>&nbsp;</td><td><b>Growth environment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Growth support&nbsp;</td><td>Soil (type, supplier), Agar 
(type, supplier), Vermiculite (type, supplier), hydroponic system (type,
 supplier, nutrient concentrations) or other support including cell 
culture (media, volume, cell number per volume).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Growth location&nbsp;</td><td>Field trial (location), 
climate chamber (size m^3), greenhouse (details on accuracy of control 
of light, humidity and temperature conditions), other location (details 
on size m^3, accuracy of control of light, humidity and temperature 
conditions).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Growth plot design&nbsp;</td><td>The way to randomize the 
different genotype-environment interactions. Either descriptive or using
 established nomenclature e.g. latin square.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Light&nbsp;</td><td>Light quality, light source model/type,
 light intensity (best reported as empirically measured at plant 
height), luminescence (daylight) period (h). For field trials: average 
light parameters in growing season. Information on time and location of 
the field trial enables tracking of more precise information if 
necessary.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Humidity&nbsp;</td><td>Humidity (percentage) at day and at 
night. For field trials: average humidity parameters in growing season. 
Information on time and location of the field trial enables tracking of 
more precise information if necessary.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Temperature&nbsp;</td><td>Temperature (°C) at day and at 
night. For field trials: average temperature (°C) at day and at night in
 growing season. Information about time and location of the field trial 
enables tracking of more precise information if necessary.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Watering regime&nbsp;</td><td>Amount and time of watering 
per day. For field trials: average rain fall in growing season. 
Information on time and location of the field trial enables tracking of 
more precise information if necessary. For hydroponic systems: frequency
 of solution change.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Nutritional regime&nbsp;</td><td>Amount and time of additional nutrients given to plants.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Date(s) of plant establishment&nbsp;</td><td>Depending on 
plant study, such dates could comprise: sowing, germination, 
transplanting, cutting, grafting or other appropriate time stamps. Plant
 development stage description should accompany time stamps using 
established nomenclature (Boyes et al. 2001; Pujar et al. 2006; 
Palmquist et al. 2006).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Other specific metadata&nbsp;</td><td>translocation of 
plants from one chamber to another, rotational schema of trays within a 
climate chamber, agrochemical or preventive maintenance information that
 is not part of 'treatment' factors.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>3</b>&nbsp;</td><td><b>Treatment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Treatment factors</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Biotic treatment&nbsp;</td><td>&nbsp;</td><td>infection (species), herbivore attack (species), competition with other plants (species)&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Abiotic treatment&nbsp;</td><td>&nbsp;</td><td>light 
intensity variations, cold acclimation (temperature), heat stress, 
drought (description of residual growth support moisture, or 
quantitative description of reduction in watering regime), water stress,
 saline stress&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Intervention treatment&nbsp;</td><td>&nbsp;</td><td>application of agrochemicals, enzyme inhibitors, hormones, elicitors&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Treatment dose or intensity levels&nbsp;</td><td>Depending on treatment factors.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Treatment time, time intervals and duration before harvest&nbsp;</td><td>Depending on treatment factors and treatment time.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>4</b>&nbsp;</td><td><b>Harvest</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Harvest date and time&nbsp;</td><td>Harvest time relative 
to the luminescence cycle. Duration of harvest if relevant to the plant 
study (e.g. for volatile analysis).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Plant growth stage&nbsp;</td><td>It is advised to refer to 
established literature, e.g. for Arabidopsis (Boyes et al. 2001) and 
Medicago truncatula (Palmquist et al. 2006); and for general growth 
stage ontology (Pujar et al. 2006).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Time after harvest before stopping cellular metabolism&nbsp;</td><td>May
 be greater than weeks for certain post-harvest physiology experiments, 
may be less than seconds for assessing high turnover metabolites.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Metabolism quenching method&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Harvest method&nbsp;</td><td>Details of operation to gather the plant organ (sample)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample storage&nbsp;</td><td>Details of pooling of plant 
tissues for analysis; operations to store sample (e.g. freeze-drying, 
grinding) prior to preparation for metabolomic analysis; duration and 
temperature of storage before extraction for analysis.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Chemical analysis</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>1</b>&nbsp;</td><td><b>Sample preparation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sampling process and protocol</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Replicate sampling and analyses&nbsp;</td><td>Substantial 
biological variance exists within all organisms; therefore replicate 
sampling and analyses are critical to provide a statistical basis for 
data evaluation and interpretation. A minimum of triplicate (n = 3) 
biological sampling is proposed with n = 5 preferred. Biological 
replicates (repetitive analyses of samples obtained from different 
individuals or pooled individuals from a population) are preferred over 
analytical replicates (repetitive analyses of the same sample obtained 
from the same individual or pooled individuals) as biological variance 
almost always exceeds analytical variance.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tissue harvesting method&nbsp;</td><td>For example, sample 
freezing method (e.g. liquid N2, dry ice and acetone bath, freeze 
clamping, etc.), sample wash method for removing unwanted external 
components, time and duration for tissue collection (e.g. time from 
tissue resection to liquid N2 freezing), temperature, and sample storage
 prior to further preparation (e.g. –80 degrees celsius for 2 weeks). 
All temperatures should be measured if possible; however temperature 
setpoints are acceptable assuming quality monitoring was performed and 
no abnormalities recorded.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Biofluid harvesting or collection method&nbsp;</td><td>For 
example, syringe, collection onto refrigerated surface, vacuum 
system/vacutainers used for blood collection, storage vessel and 
anticoagulant (if relevant), temperature, velocity and duration of 
centrifugation, and sample freezing method.&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Tissue processing method&nbsp;</td><td>For example, 
lyophilization, fresh tissue processing, pulverization/homogenization, 
tissue cell lysis (e.g. liquid N2 grinding, manual or electric 
homogenization, beadbased homogenization, ultrasonic cell lysis, buffer 
based lysis, etc.).&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions prior to extraction or further processing&nbsp;</td><td>&nbsp;</td><td>–80°C, duration, atmospheric pressure or vacuum, desiccation, preservatives added&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Relocation and shipping of tissues from one laboratory to another&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Extraction method&nbsp;</td><td>It is noted that degassing 
of solvents is important to minimize redox reactions of sensitive 
compounds such as ascorbate, cysteine, etc.&nbsp;</td><td>1 ml ice-cold 
methanol (MeOH) per 6 mg lyophilized tissue, two extractions combined, 
CHCl3/MeOH (2/1, v/v) followed by 10% trichloroacetic acid 
extraction.&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Extract concentration, dilution, and resolubilization processes&nbsp;</td><td>&nbsp;</td><td>Dried under nitrogen, resolubilized in H2O or pyridine&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Extract Enrichment</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Solid phase extraction&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Desalting, molecular weight cut-off, ion exchange, etc.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Extract cleanup and additional manipulation&nbsp;</td><td>&nbsp;</td><td>ultrafiltration, removal of paramagnetic ions, addition of metal chelators such as EDTA, citrate&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Extract storage and relocation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Storage conditions prior to and during analysis&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Relocation and shipping of extracts from one laboratory to another&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>2</b>&nbsp;</td><td><b>Chromatography</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Chromatography instrument description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer, model number, software package and version number or date.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Auto-injector</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Injector model/type, software version, injection volume, wash cycles (volumes), solvent.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Separation column and pre- or guard column</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer, model number/name, stationary media 
composition (support and coating, e.g. silica, C18, etc.) and physical 
parameters (i.e. coating thickness for GC/MS, particle size and pore 
size for LC/MS), internal diameter, and length.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Technique-specific sample preparation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Resuspension of sample (e.g. in mobile phase), amount injected.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Derivatization reaction conditions if relevant, (e.g. 
OMS/trimethylsilyl; chemical manufacturer, temperatures, and 
duration).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample spiking e.g. internal standards, retention index standards.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Separation parameters</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Method name (a detailed method can be published elsewhere 
and referenced here by a unique protocol identifier), injector 
temperature, split or splitless mode and ratio, LC post-column split, 
mobile phase compositions, mobile phase flow rates, pressure, 
thermal/solvent/solute gradient profiles.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>3</b>&nbsp;</td><td><b>Mass spectrometry</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer, model number, software package and version (name, number or date).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sample introduction and delivery</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>From GC, from LC, direct infusion without chromatography, direct infusion using dedicated autosampler flow rate.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Ionization source</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Ionization mode (EI, APCI, ESI etc.), polarity (positive or
 negative-ion analysis), vacuum pressure, skimmer/focusing lens voltages
 (e.g. capillary voltage etc.), gas flows (e.g. nebulization gas, cone 
gas etc., source temperature). Although these values will vary between 
instruments, they should provide a cumulative view of the ionization 
conditions sufficient to enable reproduction of the experiment.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Mass analyzer description and acquisition mode</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Type (quadrupole, ion-trap, time-of-flight, FT-ICR, 
including combinations of these for hybrid instruments), acquisition 
mode (full scan, MSn, SIM, MRM, etc.).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Technique-specific sample preparation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Re-suspension of sample (e.g. in MeOH:water 1:1 with 0.2% 
formic acid), derivatization, volume injected, and internal calibrant(s)
 added (if relevant).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Data acquisition parameters</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Date, operator, data acquisition rate, m/z scan range, 
compounds used for m/z calibration, mass resolution, mass accuracy, 
logic program used for data acquisition (often reported for ion-traps), 
spectral acquisition rate, vacuum pressure, and/or lock spray 
(concentration, lock mass, flow rate, and frequency).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>4</b>&nbsp;</td><td><b>Nuclear magnetic resonance spectroscopy</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer, model name/number, magnetic field strength in
 Tesla (example 14.1 T Varian Inova; 18.8 T Bruker Avance) or proton 
resonance frequency e.g. 600 MHz, and console description.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument configuration</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>VT control, pulsed field gradients (z or x,y,z) and maximum
 gradient strength (if used), number of shims, number of channels.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Probe type (e.g. 10 mm 31P, 5 mm HCN cold probe, 3 mm 
flow-probe etc.), solution or solid-state, automation or manual 
operation, autotune or manual tune, and probe gas. For LC-NMR: sample 
handler, injection volumes, wash cycles and solvent.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument-specific sample preparation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Volume, extract/powder/intact organisms, tissue or cells, 
type of NMR tube (e.g. conventional, Shigemi, microcell etc.), pH, 
solvent (D2O, CD3OD, CDCl3 etc.), buffer, chemical shift or calibration 
standard.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Data acquisition parameters</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>For 1-D 1H or X-nucleus NMR: temperature, observed nucleus,
 pulse sequence name, pulse sequence implementation (e.g. gradient 
selection, sensitivity enhancement), spin rate or statement of no spin, 
solvent saturation or decoupling method, presence or absence of 
heteronuclear decoupling (e.g. isotope-enriched samples), decoupling 
mode and bandwidth; spin lock field strength (in Hz) and duration (in 
sec), mixing time (for NOESY, ROESY etc.), spin echo time (e.g. for 
relaxation analysis or broadline suppression), RF pulse widths, any 
selective pulse shapes and durations used, magnetic field gradient pulse
 times and shapes, spectral width, acquisition time, relaxation delay 
and additional delays (mixing time, etc.), interpulse delay (or recycle 
time), digitization parameters, spectral width and acquisition time, 
number of transients, and number of steady states transients (i.e. dummy
 scans). For solvent suppression: technique, excitation maximum and 
bandwidth.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Additional parameters for 2-D and higher dimensional NMR: 
observed nucleus in F2 and F1, pulse sequence, excitation pulse widths 
for relevant nuclei, spectral width in F2 and F1, solvent saturation 
method, number of transients in t2 and number of increments in t1, 
acquisition times for t2 and t1, phase sensitive or magnitude detection.
 pulsed field gradient strengths and shapes (z or x,y,z) and maximum 
gradient strength (if relevant to the pulse sequence).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Additional parameters for X-nucleus 1D and higher 
dimensional NMR: direct or indirect detection, proton decoupling mode 
(Waltz, Garp, Wurst, Stud etc.) and effective band width, evolution time
 for constant time experiments, editing mode (cf. INEPT-based 
experiments), heteronuclear spin lock strength and mixing time (e.g. 
HCCH-TOCSY).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Additional parameters for pseudo 2D NMR experiments: 
physical parameter varied in the t1 dimension (e.g. T2, T1, diffusion 
period, chromatographic separation time as in LC-NMR, etc.), pulse 
sequence, array of values used for physical constants.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>5</b>&nbsp;</td><td><b>stable isotopes &amp; flux analysis</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Isotope labeled precursors used</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Element/isotope, position(s), percent labeled; e.g. [13C-1]-D-glucose (98%), [15N2]-L-glutamine (99%).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Isotope source (i.e. manufacturer), chemical purity of the 
labeled compound(s), concentration of the compound, fraction of total 
present (requires detailed breakdown of media composition for cell and 
tissue studies, including analysis of any added FCS or other growth 
supplements; labeling scheme).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Total number of moles isotope added during the experiment.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Duration of pulse label or continuous addition&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>6</b>&nbsp;</td><td><b>Fourier transform infrared (FT-IR) spectroscopy</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>FT-IR spectrometer instrument description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer, model number, software name and version number or date.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument configuration</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Type of sampling compartment used, including where necessary type of microscope employed.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Type of detector used (DTGS (deuterated triglycine 
sulphate), MCT (mercury cadmium telluride), and/ or FPA (focal plane 
array).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Technique-specific sample preparation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Resuspension of n mg ml–1 sample into solvent, volume analysed.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sample presentation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Transmission measurement: in KBr, or on ZnSe, Si windows.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reflectance measurement: on Si, Au, Al, or other defined metal sample carrier.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Diffuse reflectance measurement: on defined metal sample carrier.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sampling area, and for imaging pixel size.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Data acquisition parameters</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Wavenumber (cm–1) range.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Rate of acquisition.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Spectral resolution (in cm^–1).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Number of spectra co-added.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Number of data points in the resultant spectrum, and how this is displayed (absorbance or transmission).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>7</b>&nbsp;</td><td><b>Instrumental performance and method validation</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Minimum Reporting of Instrumental 
Performance Parameters is Encouraged. The nature and method(s) used to 
ensure sensitive and selective instrumental performance should be 
reported and the following details and descriptors are deemed 
appropriate.</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Mass spectrometry instrument performance validation 
parameters reported might include chemical description of the m/z 
calibration standard used, accuracy of m/z calibration, mass resolution,
 and ion source optimization parameters. For hyphenated MS methods, 
suggested reporting parameters could include chromatographic resolution,
 accuracy and precision of internal standard(s) or retention time 
markers, accuracy and precision for replicated analyses, accuracy and 
precision for validation sample(s), and cycles per 
column/injector/septum/ blank.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>NMR instrument performance verification parameters might 
include calibration standard used (name, chemical shift and 
concentration; e.g. 0.5 mM DSS or 1 mM TMS at 0.0 ppm), statement of 
line width of the standard at 50% and 1% of its full height (e.g. DSS, 
TSP or TMS methyl peak) or residual water, pH marker used (if relevant) 
and shift correction. For X nuclei: external or internal reference and 
conditions, and correction made for susceptibility effects. Reporting of
 shift referencing method for indirect dimension in 2D experiments 
(direct or indirect based on c ratios) would also be beneficial.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Quantitative Method Validation. Two methods
 of quantitative analysis are typically used in metabolomics and include
 relative and absolute quantification. Relative quantification (i.e. 
reporting of metabolite(s) instrument response relative to an internal 
standard or another metabolite(s) level such as the sum of all 
metabolite abundance) is typically used in non biased metabolomics. 
Whereas, absolute quantification (determination of the absolute 
concentration of a metabolite(s) through correlation of its instrument 
response to that of a known concentration series of the same metabolite)
 is commonly used in targeted metabolite(s) analysis.</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Relative Quantification</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>A description and quantifier of the added exongenous isotopically labeled or unlabeled metabolite(s).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>A description of the method used for assessing instrument 
response (e.g. peak integration, binning/ bucketing or deconvolution 
method, intensity normalized to reference,&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>For NMR, descriptions for correction for saturation effects
 - T1 values measured), and provide relaxation agents if added (type, 
amount). For direct X-detection (especially 13C or 31P), correction for 
nuclear Overhauser enhancement as well as saturation. For non-deuterated
 aqueous samples, state any corrections made for nonlinear excitation 
profile and method.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reporting on replicate analyses, standard error/ deviation of quantification.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Absolute Quantification</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Calibration curves should be generated for each metabolite 
to be quantified in the same biological matrix and include a sufficient 
number of standard solutions to adequately define the instrument 
response to concentration relationship (i.e. suggested minimum of at 
least one standard solution per order of change in concentration). The 
range of standard solutions used and the range of linearity with 
correlation coefficient should be reported.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>A quantifier of the method accuracy (i.e. standard 
deviation, relative standard deviation, coefficient of variance) should 
be reported and bias assessed if possible (bias; due to method, lab, ion
 suppression, etc.).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>A quantifier of the method precision (i.e. standard 
deviation, relative standard deviation, coefficient of variance) should 
be reported.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>The lower limit of quantification (LLOQ) and confidence 
level should be reported. The LLOQ is defined as the minimum 
concentration generating an instrumental signal-to-noise response ratio 
of 10. The LLOQ has alternatively been defined as 5 times the limit of 
detection (LOD). The LOD is defined as the concentration that yields a 
minimum instrumental signal-to-noise ratio of 3.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Additional quantitative descriptions of recovery and/or stability provide additional method validation.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>8</b>&nbsp;</td><td><b>Data preprocessing</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Post Acquisition Data Pre-processing</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Data file format used and/or conversion methods should be 
reported. Examples include conversion of proprietary file formats to 
more universal formats such as net.cdf, XML, MZmine, etc.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Details of any data pre-processing methods 
which convert raw instrumental data into organized or tabular file 
formats should be reported.</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Examples for MS might include: background subtraction, 
noise reduction, curve resolution for temporal chromatographic 
alignment, peak picking, peak thresholding, spectral deconvolution, 
and/or metabolite identifications. Some comparative methods do not 
resolve or identify individual metabolites prior to comparative 
analysis. The general experimental details describing these methods 
should still be reported and should be sufficient so that others can 
replicate the data processing.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Examples for NMR data pre-processing might include 
phase-correction method (e.g. automatic, manual), conversion from time 
to frequency domain (e.g. Fourier Transform), degree of zero filling, 
degree of linear prediction; apodization parameters and window functions
 in all dimensions (exponential, Gaussian, sine bell etc.), baseline 
corrections (dc offset, linear or non-linear corrections), first point 
multipliers, any shifting of the free induction decays.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>For data analysis of isotope labeling of flux experiments, 
the method for determining positional and fractional labeling, standard 
error of the estimates; and estimated isotope recovery in observable 
fractions (and fraction of total isotope supplied) should be 
described.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Examples for FT-IR spectroscopy might include conversion 
from time to frequency domain (e.g. Fourier Transform), and degree of 
zero filling. Baseline corrections parameters might include offsets, 
level and type of derivatisation (including algorithm, window size for 
smoothing), and whether or not CO2 was removed from spectra (deleted or a
 linear trend fitted).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b>9</b>&nbsp;</td><td><b>Identification of metabolites</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Non-novel metabolites</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>A minimum of two independent and orthogonal data relative 
to an authentic compound analyzed under identical experimental 
conditions are proposed as necessary to validate non-novel metabolite 
identifications (e.g. retention time/index and mass spectrum, retention 
time and NMR spectrum, accurate mass and tandem MS, accurate mass and 
isotope pattern, full 1H and/or 13C NMR, 2-D NMR spectra). The use of 
literature values reported for authentic samples by other laboratories 
are generally believed insufficient to validate a confident and rigorous
 identification. The use of literature or external laboratory data 
result in level 2 identifications.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>If spectral (MS or NMR) matching is utilized in the 
identification process then the authentic spectra used for the spectral 
matching should be described appropriately or libraries made publicly 
available. It is preferred that the reference spectra are made available
 at no cost, but the CAWG recognizes that this may not always be 
possible for commercialized libraries (NIST, Wiley, etc.). However, the 
premise of this minimum is that authors document and provide the 
spectral evidence to validate the metabolite identifications. If the 
authors choose not to provide the experimental evidence to support the 
identifications, then the identifications should be reported as 
‘putative identifications’.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Metabolite identifications based upon additional orthogonal
 data (i.e. more than two) are highly advantageous, provide additional 
confidence, and are often necessary to provide unambiguous 
identification of stereo configuration. Additional data consistent with 
best chemical practices might include: selective solvent extraction, 
retention time, m/z, photodiode array spectra, kmax and emax,chemical 
derivatization, isotope labeling, 2D NMR, IR spectra, etc.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Novel metabolites</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Describe extraction, isolation, and purification followed 
by elemental analysis, accurate mass measurement, ion mass fragmentation
 patterns, NMR (1H, 13C, 2D), and other spectral data such as IR, UV, or
 chemical derivatization.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Unknown metabolites</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>For NMR, the exact chemical shift and multiplicity of at 
least one nucleus in the metabolite should be part of the unknown 
nomenclature For example, an unidentified triplet at 1.16 ppm could be 
reported as: ‘unknown (1.16 ppm, triplet)’. When such a signal can be 
correlated with other atoms in the same molecule using multidimensional 
or multi-pulse techniques, the chemical shifts and the connectivity of 
such correlated nuclei in the unknown should be reported in the work. In
 such cases the molecular fragement may be identified, such as 
‘isopropyl group’.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>For MS, the retention time, retention index, and/or 
prominent ions in the mass spectrum should be reported along with MS-MS 
data if available (also see Bino et al. 2004).&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Xenobiotics (e.g. administered drugs, related drug 
metabolites) or other exogenous compounds such as herbicides, 
pesticides, etc. should be rigorously distinguished from endogenous 
metabolites for all unknowns and if possible.&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>NMR spectroscopy</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sample description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Reference to biological sample provenance details&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>pH of biological sample&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>pH of sample after buffer has been added&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Field frequency lock</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Chemical name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Additional solute</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Chemical name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Concentration in sample&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Solvent</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Chemical name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Concentration in sample&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Chemical shift standard</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Chemical name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Concentration in sample&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Concentration standard</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Chemical name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Internal or external&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Concentration in sample&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Geographical location of the instrument&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Magnet</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serial number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Model&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Field strength&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Probe</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serial number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Model&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Gradient strength&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Console</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serial number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Model&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Acquisition computer</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serial number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Model&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Operating system and version number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Application software and version number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Autosampler</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Serial number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manufacturer&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Model&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Application software and version number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Acquisition parameters</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Parameters recorded once for each sample</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Acquisition parameters file reference&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Shaped pulse file reference&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Sample details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample introduction method&nbsp;</td><td>&nbsp;</td><td>tube, flow probe, rotor&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Size of tube or flow probe or rotor&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample temperature in autosampler&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample temperature in magnet&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument operation details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Sample spinning rate&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Water suppression technique&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Pulse sequence name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Pulse sequence file reference&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Pulse sequence literature reference&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Data acquisition details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Number of steady-state scans&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Number of scans&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Relaxation delay&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Parameters recorded once for each NMR analysis dimension</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Instrument operation details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Irradiation frequency&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Acquisition nucleus&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>90° pulse width of irraditation nucleus&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Data acquisition details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Dwell time&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Number of data points acquired&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Parameters recorded for higher dimensions</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Name of encoding scheme&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Hadamard frequency&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Quality control</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Identity of signal within output data&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Linewidth of signal prior to window function processing&nbsp;</td><td>Full width at half-maximum (FWHM)&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Peak width at five percent total height&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>FID and spectral processing parameters</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Parameters recorded once when a raw data set is processed</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Post-acquisition water suppression method name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Time-based to frequency-based data transformation method&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Name of chemical used to reference the spectrum&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Processing software details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Processing software name and version number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Spectral projection details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Method of projection&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Projection axis&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Parameters recorded once for each NMR analysis dimension</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Processing parameters file reference&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Number of data points in spectrum&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Zero order phase correction&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>First order phase correction&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Calibration reference shift&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Baseline correction method&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Spectral de-noising method&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Window function details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Window function name&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Function parameter and value&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Parameters recorded for the second dimension only</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>2D J-resolved processing details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Flag to indicate if the data has been tilted&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Flag to indicate if the data has been symmetrised&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Spectral quantitation parameter set</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Quantitation type&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Quantitation algorithm&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Quantitation parameters&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Manual quantitation description&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Processing software details</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Processing software name and version number&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="#DDDDDD">
<td><b></b>&nbsp;</td><td><b>Analysis description</b>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Date and time of data acquisition&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Institution&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Operator&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
<tr bgcolor="FFFFBB">
<td></td><td>Supervisor&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td>
</tr>
</tbody></table>
</td>
</tr>
</tbody></table>
<br>
<hr>
<p>
<b>&nbsp;Sources</b>
</p>
<blockquote>
<p>
<tt>CIMR</tt>
</p>
</blockquote>
<hr>
<p>
<b>&nbsp;Module notes</b>
</p>
<blockquote>
<p>&nbsp;</p>
</blockquote>
<hr>


</body></html>